src/ship_cmd.cpp

```diff
@@ -549,6 +550,32 @@ static void ShipController(Ship *v)
 
 	v->ShowVisualEffect();
 
+	/* Rotating on spot */
```

Comment:
Would this also work? I think it is more concise:
if (v->direction != v->rotation) {
if (tick_counter) {
DirDiff diff = dirdifference...
v->rotation = ChangeDir(v->rotation, diff > DIRDIFF_REVERSE ? 45_LEFT : 45_RIGHT);
updateviewport
}
return;
}

Reply:
Seems reasonable, and means the DIRDIFF enum doesn't need to be modified.